Cargo.toml

```diff
@@ -28,6 +28,9 @@ tokio = { version = "1.17.0", features = ["rt-multi-thread", "io-util", "macros"
 tracing = "0.1.32"
 tracing-subscriber = "0.3.10"
 uuid = { version = "0.8.2", features = ["serde", "v4"] }
+tokio-util = {version = "*", features = ["codec"]}
+futures = "0.3.21"
+# bytes= "1.1.0"
```

Comment:
Could you remove the `bytes` line, use a real non-wildcard version of `tokio-util` (otherwise crates.io will refuse to publish bore), and use the `futures-util` crate rather than the entire `futures` crate, which is large?

<img width="740" alt="image" src="https://user-images.githubusercontent.com/7550632/163890817-80dbf911-ccb7-4097-8af1-3d03cdea1b24.png">